coronavirus/coronavirus.py

```diff
@@ -834,11 +835,20 @@ def get_compare_data(countrynames, rolling=7):
     df_c = pd.DataFrame()
     df_d = pd.DataFrame()
 
-    for countryname in countrynames:
-        c, d = get_country(countryname)
+    if regions:
```

Comment:
The implicit assumption here is that if `regions` is not None, then this is a about Spain. This may change over time. I suggest to add an if statement along the lines of `if countryname=='Spain' and region is not None` so that this can be extended in the future for other countries.